docker/bash_bootstrap.sh

```diff
@@ -1,40 +0,0 @@
-#!/bin/bash
-
-# Copyright (C) 2021 BMW CarIT GmbH
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-# SPDX-License-Identifier: Apache-2.0
-# License-Filename: LICENSE
-
-# We will parse the necessary shells modules for each language
-# independently
-
-set -e
-
-# Global functions
-add_local_path () {
```

Comment:
my changes dropped `add_local_path` even though that is still used in the other .sh files

Reply:
I am working on a fix

Reply:
fix: https://github.com/heliocastro/ort/pull/19

Reply:
I merged your PR, thanks